dss-api.yml

```diff
@@ -126,6 +126,73 @@ definitions:
       - uuid
       - name
       - versions
+  blob_common_metadata:
+    type: object
+    description: Common fields between the files list of a bundle and the metadata tracking an unique file.
+    properties:
+      uuid:
+        type: string
+        description: File unique ID
+      timestamp:
+        type: integer
+        format: int64
+        description: Timestamp of file creation measured in milliseconds since epoch
```

Comment:
Doesn't ISO 8601 support milliseconds with an optional decimal? https://www.w3.org/TR/NOTE-datetime seems to indicate that there can be a variable number of fractional digits after the optional dot. [Also found this](https://nbsoftsolutions.com/blog/iso-8601-and-nanosecond-precision-across-languages). I can't see the actual standard since it seems you have to pay money for it (what a mess).

So IMO that shouldn't be the deciding factor. I'm leaning in favor of ISO timestamps.